client/src/main.js

```diff
@@ -13,15 +14,12 @@ Vue.config.productionTip = false
 import App from './App.vue'
 
 Vue.use(BootstrapVue)
+Vue.use(VueCookies)
 Vue.config.productionTip = false
 Vue.prototype.$apiURL = 'http://sagelysown.owlinventory.com/api/'
 Axios.defaults.baseURL = 'http://sagelysown.owlinventory.com/'
 Axios.defaults.withCredentials = true;
 Vue.prototype.$http = Axios;
-const token = localStorage.getItem('token')
-if (token) {
-  Vue.prototype.$http.defaults.headers.common['Authorization'] = token
-}
```

Comment:
The browser will automatically send the cookie with the JWT to the api with each request.